XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/Hands/BaseHandControllerDataProvider.cs

```diff
@@ -0,0 +1,342 @@
+                    {
+                        if (rightHand == null)
+                        {
+                            rightHand = hand;
+                        }
+                    }
+                }
+            }
+
+            if (leftHand != null)
+            {
+                foreach (var fauxJoint in leftHandJointTransforms)
+                {
+                    if (leftHand.TryGetJointPose(fauxJoint.Key, out MixedRealityPose pose))
+                    {
+                        fauxJoint.Value.SetPositionAndRotation(pose.Position, pose.Rotation);
+                    }
+                }
+            }
+
+            if (rightHand != null)
+            {
+                foreach (var fauxJoint in rightHandJointTransforms)
+                {
+                    if (rightHand.TryGetJointPose(fauxJoint.Key, out MixedRealityPose pose))
+                    {
+                        fauxJoint.Value.SetPositionAndRotation(pose.Position, pose.Rotation);
+                    }
+                }
+            }
+        }
+
+        /// <inheritdoc />
+        public bool TryGetJointTransform(TrackedHandJoint joint, Handedness handedness, out Transform jointTransform)
+        {
+            Dictionary<TrackedHandJoint, Transform> fauxJoints;
+            IMixedRealityHandController hand;
+
+            if (handedness == Handedness.Left)
+            {
+                hand = leftHand;
+                fauxJoints = leftHandJointTransforms;
+            }
+            else if (handedness == Handedness.Right)
+            {
+                hand = rightHand;
+                fauxJoints = rightHandJointTransforms;
+            }
+            else
+            {
+                jointTransform = null;
+                return false;
+            }
+
+            if (fauxJoints != null && fauxJoints.TryGetValue(joint, out Transform existingJointTransform))
+            {
+                jointTransform = existingJointTransform;
+                return true;
+            }
+
+            Transform newJointTransform = new GameObject().transform;
+            newJointTransform.name = $"Joint Tracker: {joint} {handedness}";
+
+            // Since this service survives scene loading and unloading, the fauxJoints it manages need to as well.
+            Object.DontDestroyOnLoad(newJointTransform.gameObject);
+
+            if (hand != null && hand.TryGetJointPose(joint, out MixedRealityPose pose))
+            {
+                newJointTransform.SetPositionAndRotation(pose.Position, pose.Rotation);
+            }
+
+            fauxJoints.Add(joint, newJointTransform);
+            jointTransform = newJointTransform;
+            return true;
+        }
+
+        /// <inheritdoc />
+        public bool IsHandTracked(Handedness handedness)
+        {
+            switch (handedness)
+            {
+                case Handedness.None:
+                    return leftHand == null && rightHand == null;
```

Comment:
This is weird. I think if you say none, it's because you don't expect there to be tracked hands - and this would return true if there are no tracked hands, which doesn't make a whole lot of sense.

Reply:
IsHandTracked(Handedness.None) would return true, if no hands are tracked. I can see how this is confusing.

Reply:
Maybe add a comment to the case to explain